### DISCUSSION TITLE
Question: can you use this program without wifi?

### DISCUSSION THREAD

**Discussion by iowasurvivor** [outdoorbits/little-backup-box#174] [General]
I'm a bit of a n00b, so pardon if this is a simple question. 

I'm looking to use my Pi as a photo backup while traveling (Camera/SD card to USB). I set it up per the documentation, and I can control it my phone. And that's great.

However, what I'd like to use this pi without a wifi connection. I want to get a 5" screen to plug into my pi and show the program's UI without a phone or a wireless connection. 

Either I'm doing it wrong or the backup box can't be controlled without a phone/wireless connection. Help?

**Comment by outdoorbits** [comment]
LBB cannot use a display in this way, at least not yet. I toyed with the 
idea. But there were two reasons against it:

1. The display would require significant additional energy.
2. The display would probably always be significantly worse than that of 
a halfway up-to-date smartphone.

In principle, however, it shouldn't be too difficult to implement: 
Instead of the Raspberry Pi OS Lite, you would have to install the 
desktop version. From here you could then use the browser.

**Reply by outdoorbits** [reply]
But you can configure it to start in a defined backup mode and shutdown when finished. If you use a little and cheap display as described in the wiki, you can have a look on the progress.
